### ISSUE TITLE
Mapping Exception with property set to Ignore

### ISSUE THREAD

**Issue by rafaelbelo** [TinyMapper/TinyMapper#35]
In the source, the property Anexo is a string, in the target, it is an HttpPostedFileBase;

This is my code:

Relato relato = relatoService.BuscarPorProtocolo(protocolo);

Nelibur.ObjectMapper.TinyMapper.Bind<Relato, AcompanhamentoRelatoViewModel>(config => config.Ignore(x => x.Anexo));
AcompanhamentoRelatoViewModel acompanhamento = Nelibur.ObjectMapper.TinyMapper.Map<AcompanhamentoRelatoViewModel>(relato);

The last line throws an exception of type System.InvalidCastException
Unable to cast object of type 'System.String' to type 'System.Web.HttpPostedFileBase'.

This is the only property of this type, so somehow I think it is ignoring the Ignore config?

**Comment by GSerjo** [comment]
Could you please share your `Relato`,  `AcompanhamentoRelatoViewModel` classes.  Looks like you have the same name property, but in one class it's a `string` and another one it's an `object`

**Comment by rafaelbelo** [comment]
Precisely. Like the exception states, one property is a string, the other is an HttpPostedFileBase. If it can't map it, then i tried to set TinyMapper to ignore it. But it seems to be ignoring my ignore statement.

`    public class AcompanhamentoRelatoViewModel
    {
        [ScaffoldColumn(false)]
        public int RelatoId { get; set; }

        [DataType(DataType.Text), DisplayFormat(DataFormatString = "{0:000000}")]
        public int Protocolo { get; set; }

        [Display(Name = "Data de Criação"), DataType(DataType.DateTime)]
        public DateTime Criacao { get; set; }

        [ScaffoldColumn(false)]
        public string NumeroIp { get; set; }

        [Display(Name = "Anexar um arquivo (zip)")]
        public HttpPostedFileBase Anexo { get; set; }

        [Display(Name = "Situação Atual")]
        public EstadoRelato Estado { get; set; }

        [DataType(DataType.MultilineText)]
        [Display(Name = "Descrição")]
        public string TextoAcompanhamento { get; set; }

        [ScaffoldColumn(false)]
        public bool Desativado { get; set; }

        public List<HistoricoRelatoViewModel> HistoricoRelatos { get; set; }
        
    }`

`    public class Relato
    {
        public virtual int Id { get; set; }
        public virtual int Protocolo { get; set; }
        public virtual int? TipologiaId { get; set; }
        public virtual int? LocalidadeId { get; set; }
        public virtual int IdentidadeId { get; set; }
        public virtual string TextoRelato { get; set; }
        public virtual string NumeroIp { get; set; }
        public virtual string Anexo { get; set; }
        public virtual EstadoRelato Estado { get; set; }
        public virtual DateTime Criacao { get; set; }
        public virtual string Status { get; set; }
        public virtual bool Desativado { get; set; }        
        public virtual Localidade Localidade { get; set; }
        public virtual Tipologia Tipologia { get; set; }
        public virtual Identidade Identidade { get; set; }
        public virtual List<HistoricoRelato> HistoricoRelatos { get; set; }
    }`

**Comment by derekforeman** [comment]
I'm seeing this as well. 
`
public class Source {
  public Guid Id {get; set;}
}

public class Target
{
  public int Id {get; set;}
  public Guid TargetId {get; set}
}

TinyMapper.Bind<Source, Target>(config =>
{
    config.Bind(o => o.Id, o => o.TargetId);
});

//Errors on the mapping with invalid type cast.


`
